src/components/Container/Container.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from "styled-components";
+import { HTMLAttributes } from "react";
+import { Orientation } from "@/components";
+
+type AlignItemsOptions = "start" | "center" | "end";
+type GapOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
+type GrowthOptions = "0" | "1" | "2" | "3" | "4" | "5" | "6";
+type JustifyContentOptions =
+  | "flex-start"
+  | "flex-end"
+  | "center"
+  | "space-between"
+  | "space-around"
+  | "space-evenly"
+  | "start"
+  | "end"
+  | "left"
+  | "right";
+type PaddingOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
+
+export interface ContainerProps extends HTMLAttributes<HTMLDivElement> {
+  alignItems?: AlignItemsOptions;
+  children?: React.ReactNode;
+  fillWidth?: boolean;
+  gap?: GapOptions;
+  grow?: GrowthOptions;
+  hasBorder?: boolean;
+  isResponsive?: boolean;
+  justifyContent?: JustifyContentOptions;
+  orientation?: Orientation;
+  padding?: PaddingOptions;
+}
+
+const Container = ({
+  alignItems = "center",
+  children,
+  fillWidth = false,
+  gap = "none",
+  grow = "1",
+  hasBorder,
+  isResponsive,
+  justifyContent = "start",
+  orientation = "horizontal",
+  padding = "md",
+}: ContainerProps) => {
```

Comment:
Should have ...props and add this to the wrapper

Reply:
Done ✅